reviews/2022-openservicemesh-annual.md

```diff
@@ -0,0 +1,94 @@
+- [SMI Controller SDK](https://github.com/servicemeshinterface/smi-controller-sdk) - a successor to smi-sdk-go that provides a simple abstraction layer for making your mesh SMI-compliant
+- [Istio SMI Controller](https://github.com/servicemeshinterface/istio-smi-controller) - a successor to the SMI istio adapter that uses the smi-controller-sdk
+
+## Year in Review
+
+- Spec stabilization and augmentation - [with current status viewable on the repo](https://github.com/servicemeshinterface/smi-spec#service-mesh-interface-documents)
+  - 2022 was a year of maturation and stability for SMI. After releasing v0.6 of the specification, we turned our attention to ensuring that the spec had time to breathe (so to speak) and that the community had time to adopt the changes. This resulted in fruitful launches of collaborations such as [SMI Conformance Testing GitHub Action](https://github.com/layer5io/meshery-smi-conformance-action), and [Service Mesh Performance](https://smp-spec.io/). These efforts didn’t stop real progress in the spec itself, as issues and features continued to be discussed, triaged, and implemented via the bi-weekly community calls and various ad-hoc design documents. 
+- Ongoing involvement with the community: CNCF slack and an open community call every other week.
+  - SMI blog with [community contribution](https://smi-spec.io/blog/pipelining-service-mesh-specifications/)
+  - New community members joining calls regularly: [communications links](https://github.com/servicemeshinterface/smi-spec#communications)
+
+- [SMI Conformance Testing blog post](https://smi-spec.io/blog/validating-smi-conformance-with-meshery/)
+
+## Annual Review Items
+
+- *Include a link to your project’s devstats page. We will be looking for signs of consistent or increasing contribution activity. Please feel free to add commentary to add colour to the numbers and graphs we will see on devstats.*
+  - Year in review for the main [SMI devstats dashboard](https://smi.devstats.cncf.io/d/8/dashboards?orgId=1&from=now-365d&to=now-1h&refresh=1d)
+    - This shows a sustained, consistent level of contributions to the spec with changes based on discussion between maintainers and the community (with an expected dropoff around KubeCon events and the holidays). While devstats can show releases, repo activity is not the only metric for a specification. It’s useful to consider nuance in the content of the updates (specifically, how it’s been clarified due to direct input from implementers.)
+  - Participation from [various organizations](https://smi.devstats.cncf.io/d/5/companies-table?orgId=1)
+
+- *How many maintainers do you have, and which organisations are they from? (Feel free to link to an existing MAINTAINERS file if appropriate.)*
+  - SMI currently has [12 core maintainers](https://github.com/servicemeshinterface/smi-spec/blob/main/CODEOWNERS)
```

Comment:
Project needs to evaluate and invoke inactive maintainers governance.

Reply:
Sure thing, but that hasn't happened yet, so this report is stating where we're at currently! Thanks.